src/azul/terraform.py

```diff
@@ -261,6 +261,18 @@ def _adjust_name_tag(resource_type: str, tags: dict[str, str]) -> dict[str, str]
     }
 
 
+def provider_fragment(region: Optional[str]) -> JSON:
+    """
+    Return a fragment of Terraform configuration JSON that specifies a
+    resource's provider. Empty JSON will be returned if the resource's region
+    is the same as the default region.
```

Comment:
Should mention that a non-default region must be configured by adding a matching provider for that region in providers.tf